README.md

```diff
@@ -0,0 +1,137 @@
+[![banner](https://raw.githubusercontent.com/nevermined-io/assets/main/images/logo/banner_logo.png)](https://nevermined.io)
+
+# DeFi Data Marketplace
+
+- [DeFi Data Marketplace](#DeFi-data-marketplace)
+  - [Get Started](#get-started)
+    - [Starting the environment](#starting-the-environment)
+    - [Configuration](#configuration)
+      - [Environment Variables](#environment-variables)
+      - [Connecting to local NVM](#connecting-to-local-nvm)
+      - [Connecting to Mumbai](#connecting-to-mumbai)
+  - [Testing](#testing)
+  - [Code Style](#code-style)
+  - [Production](#production)
+  - [Releases](#releases)
+  - [Changelog](#changelog)
+  - [Build](#build)
+  - [Attribution](#attribution)
+  - [License](#license)
+
+## Get Started
+
+This repo contains a client and a server, both written in TypeScript:
+
+- **client**: Nextjs app setup with [Nevermined SDK JS](https://github.com/nevermined-io/sdk-js). Web app with the markeplace frontend.
+- **server**: Node.js app, using [Express](https://expressjs.com). Used to create bundles on user purchase.
+
+To run these services first it's needed to install the dependencies in each folder with
+
+```bash
+npm install
+```
+
+Then to run the client in development mode should run
+```bash
+npm run dev
+```
+
+And for the server
+```bash
+npm run start-watch
+```
+
+Open [http://localhost:3000](http://localhost:3000) to view the client in the browser. 
+
+### Starting the environment
+
+The client is configured to run as a standalone service, using the configuration provided, the client will connect to the test environment in mumbai testnet. In order to change this configuration and use a local nevermined stack change the enviroment variables described in the section [environment variables](#environment-variables)
+
+The server needs a database where save and retrieve the data needed to create the bundles, To create this database a docker-compose.yml file is provided. To create the database once that we are in the server folder we can run 
+
+```bash
+docker-compose up
+```
+
+This will create a MySql database where the server will store the data, but first we need to create the database schema.To do so, we can execute [this script](https://github.com/nevermined-io/defi-dataset-loader/blob/master/data/database_schema.sql) in the new database created, that will create the schema for us. 
```

Comment:
The first sentence is a bit confusing. It's a bit like you're saying that we need to create the schema before we create the db which doesn't make much sense. Maybe do
```
server will store the data. Next step is to create the database schema.
```